pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -0,0 +1,123 @@
+{ stdenv
+, rustPlatform
+, fetchFromGitHub
+, fetchurl
+, maturin
+, pip
+, pytest
+, python
+, pythonRecompileBytecodeHook
```

Comment:
```suggestion
, isPy3k
, pythonRecompileBytecodeHook
```

Reply:
`distabled` is an attribute accepted by `buildPythonPackage`, not of `buildRustPackage`.

Reply:
Ah, I assumed it was universal in ``makeDerivation``. Thanks for the catch.